js/binaryCalculator.js

```diff
@@ -173,6 +173,17 @@ document.addEventListener('keydown', event => {
     }
 });
 
+function decimal(){
+  console.log("decimal" + res.value);
+  let binaryString = res.value;
+  if (/^[01]+$/.test(binaryString)) {
+    res.value = parseInt(binaryString, 2);
+    console.log("decimal" + res.value);
+    console.log("decimal" + parseInt(binaryString, 2));
+  } else {
+    res.value = "Invalid binary input";
+  }
+}
```

Comment:
I think the update will break the calculator if the decimal value is displayed in the calculator. Can you make the display on popup modal or something else that not break the calculator functionality ?

And also you can remove the console.log line code

Reply:
I will do that